rdf_differ/diff_getter.py

```diff
@@ -0,0 +1,242 @@
+"""
+diff_getter.py
+Date:  23/07/2020
+Author: Eugeniu Costetchi
+Email: costezki.eugen@gmail.com 
+"""
+import json
+import urllib
+from typing import List, Tuple
+from abc import ABC, abstractmethod
+
+import requests
+from SPARQLWrapper import SPARQLWrapper, JSON
+from requests.auth import HTTPBasicAuth
+
+
+class AbstractDiffGetter(ABC):
+    """
+        An abstract class that return information about the available diffs.
+    """
+
+    @abstractmethod
+    def list_datasets(self) -> List[str]:
+        """
+            Return a list of available dataset names
+        :return:
+        """
+
+    @abstractmethod
+    def diff_description(self, dataset_name: str) -> Tuple[str, str]:
+        """
+            Return the dataset description.
+        :type dataset_name: the name of the desired dataset
+        :return:
+        """
+
+    @abstractmethod
+    def count_deleted_triples(self, dataset_name: str, old_version_id: str, new_version_id: str) -> int:
+        """
+            Return the number of triples that have been deleted in the new version of the dataset.
+        :type dataset_name: the name of the desired dataset
+        :param old_version_id:
+        :param new_version_id:
+        :return:
+        """
+
+    @abstractmethod
+    def count_inserted_triples(self, dataset_name: str) -> int:
+        """
+            Return the number of triples that have been inserted in the new version of the dataset.
+        :type dataset_name: the name of the desired dataset
+        :return:
+        """
+
+
+SKOS_HISTORY_PREFIXES = """
+prefix skos-history: <http://purl.org/skos-history/>
+prefix dc: <http://purl.org/dc/elements/1.1/>
+prefix dcterms: <http://purl.org/dc/terms/>
```

Comment:
I think since there are several more of these definitions, it would help with the readability and overall structure of the code to move them to a separate file.